prov/ansible/execution.go

```diff
@@ -61,6 +61,7 @@ host_key_checking=False
 timeout=30
 stdout_callback = yaml
 retry_files_save_path = #PLAY_PATH#
+nocows = 1
```

Comment:
Hum looks like very depend to your env :laughing:

Reply:
Yep, but we don't support cows in Ansible stdout handler. And in some distribs, I'm afraid it's the default behavior ?
So let's disable it.

Reply:
That's a pity but ok :smiley: